en/full-node.md

```diff
@@ -220,6 +220,24 @@ have an easy-to-use node.
 
 <div class="toccontent-block boxexpand expanded" markdown="1">
 
+## Initial Block Download(IBD)
+
+[Initial block download](/en/developer-guide#initial-block-download)
+refers to the process where nodes synchronize themselves
+to the network by downloading blocks that are new to them.
+This will happen whenever a new or offline node starts up.
```

Comment:
IBD is a specific mode of operation for BItcoin Core that does various things to prioritize download of new blocks.  For example, it doesn't accept incoming connections, doesn't request unconfirmed transactions it hears about from other nodes, and uses the memory usually dedicated to the mempool to extend the UTXO cache.

Because those are significant changes to the operating mode, Bitcoin Core only enters IBD mode if it detects that it is 100 or more blocks behind the tip of the chain---not just whenever an offline node starts up.